Azure/sonic-buildimage
# [docker-frr] Fix the start.sh where it fails in case no WARM_RESTART in configDB

[docker-frr] Fix the start.sh where it fails in case no WARM_RESTART in configDB

Signed-off-by: Zhenggen Xu <zxu@linkedin.com>

This should be back-ported to 201911 branch too.


**- What I did**
If the configDB has no warm-restart configurations, the start.sh in bgp docker would fail due to python exception as below:
```
Traceback (most recent call last):
  File "/usr/local/bin/sonic-cfggen", line 322, in <module>
    main()
  File "/usr/local/bin/sonic-cfggen", line 300, in main
    print(template.render(data))
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'WARM_RESTART' is undefined
```

**- How I did it**
Check the WARM_RESTART table first.  If does not exist, won't check the bgp_eoiu info to avoid crash.

**- How to verify it**
After the fix:
```
without any warm restart configured or if we do "sudo config warm-restart bgp_eoiu false"
bgp docker will not start the bgp_eoiu_marker

if we do:  "sudo config warm-restart bgp_eoiu true"
We can see bgp_eoiu_marker started.
```

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
